201-vmss-azureautomationdsc/README.md

```diff
@@ -0,0 +1,62 @@
+
+# VM Scale Set Configuration managed by Azure Automation DSC
```

Comment:
Please rename to 201-vmss-automation-dsc so that the name fits inside the Github folder character limit